install_template/templates/products/edb-postgres-advanced-server/base.njk

```diff
@@ -1,13 +1,17 @@
 {% extends "platformBase/" + platformBaseTemplate + '.njk' %}
 {% set packageName %}edb-as<xx>-server{% endset %}
 {% block installCommand %}
-{{super()}}
-Where `<xx>` is the version of the EDB Postgres Advanced server you are installing.
+
```

Comment:
Line 4 to 8 do not need to be added in product base file as block installcommand is already in all the respective platformBase/<respective_platform_name_files>. This is duplicating the content and also changing the option `-n` to `-y` for SUSE platforms. I think we need to revert this change and add only option `-y` to the respective platformBase/<respective_platform_name_files>. For example platformBase/debian.njk, platformBase/ubuntu.njk